packages/cli/templates/default-app/plugins/welcome/package.json.hbs

```diff
@@ -6,6 +6,7 @@
   "private": true,
   "scripts": {
     "build": "backstage-cli plugin:build",
+    "start": "backstage-cli plugin:serve",
```

Comment:
isn't this `app:serve`? 🤔

Reply:
It's the welcome plugin in the scaffolded app

Reply:
ah pssh i am blind